src/it/java/net/spy/memcached/TestConfig.java

```diff
@@ -87,6 +95,7 @@ public static engineTypeEnum fromString(String version) {
   public static final String MEMCACHED_PATH = System.getProperty(SERVER_BIN, "/usr/bin/memcached");
   public static final String MEMCACHED_VERSION = System.getProperty(SERVER_VERSION, "1.4.24");
   public static final String MEMCACHED_TYPE = System.getProperty(SERVER_TYPE, "oss");
+  public static final String MEMCACHED_CERT = System.getProperty(SERVER_CERT, "not specified");
```

Comment:
I suggest not using `server_cert` as the argument here if you are not really using it to instantiate the client. As you don't really need it, I would suggest a simple boolean parameter for `TLS_ENABLED` or something.

Reply:
We will use server_cert to start servers later